.node-version

```diff
@@ -0,0 +1 @@
+20.13.1
```

Comment:
This bumps the node version that cloudflare pages uses. It's a shame we can't comment this file. :/